### DISCUSSION TITLE
Broad Overview

### DISCUSSION THREAD

**Discussion by user8694** [EsratMaria/Reinforcement-Learning_for_Energy_Minimization_Using_CLoudsim#5] [Q&A] [answered]
Where can I learn a bit more about this approach except for the paper? The paper is great but a bit of an in-depth explanation would have been nicer and a great start for me.

**Comment by EsratMaria** [comment, marked as answer]
Hey, thank you for your question.
you can learn a bit more from this presentation file that I have [here](https://github.com/EsratMaria/Reinforcement-Learning_for_Energy_Minimization_Using_CLoudsim/blob/master/Final_thesis_Presentation.pdf). I hope it helps :)

**Reply by user8694** [reply]
Great! Thanks 🙏🏽
